ui/react-components/components/AppointmentListSidePanelTree/AppointmentListSidePanelTree.spec.jsx

```diff
@@ -0,0 +1,15 @@
+import { fireEvent, render, getByTestId } from "@testing-library/react";
+import React from "react";
+import { renderWithReactIntl } from "../../utils/TestUtil";
+import AppointmentListSidePanelTree from "./AppointmentListSidePanelTree";
+import "@testing-library/jest-dom/extend-expect";
+import { renderIntoDocument } from "react-dom/test-utils";
+
+describe("AppointmentListSidePanelTree", () => {
```

Comment:
It does not cover all the test cases as part of the AppointmentListSidePanelTree.jsx, please write more test cases ensuring if code is changed the tests would fail.

Reply:
Added more test cases